lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -254,6 +255,42 @@ def revision(scm, target, ref)
   end
 end
 
+desc "Rebuild .fixture.yml from metadata.json file"
+task :fixture_rebuild do
+  begin
+    fixtures = YAML.load_file('.fixtures.yml')['fixtures']
+  rescue Errno::ENOENT
+    fail "Unable to load .fixtures.yml"
+  end
+  
+  begin
+    metadata = JSON.parse(File.read('metadata.json'))
+  rescue Errno::ENOENT
+    fail ".fixtures.yml does not exist"
+  rescue Errno::EACCES
+    fail "Unable to read .fixtures.yml"
```

Comment:
Can you clarify what you mean that error message is wrong and which one(s)? I did a moderate amount of testing with moving .fixtures.yml out of the way and setting perms/owner to validate the error messages.